### DISCUSSION TITLE
Selected and child comments on old.reddit.com are white background.

### DISCUSSION THREAD

**Discussion by deten** [darkreader/darkreader#13086] [General]
Here is a parent comment, and its child comments below which are all white background:
![image](https://github.com/user-attachments/assets/1e36acf8-198a-4243-9f9b-1b3986758c65)

Here is a parent comment after clicking on it:
![image](https://github.com/user-attachments/assets/b68e6df4-6524-486b-99c3-588b24088949)

Any way to fix this so they remain dark?

**Comment by deten** [comment]
For clarity this is firefox browser, latest version on windows 10

**Reply by victor6267** [reply]
I'm running into a similiar situation. Edge and windows 11.

**Comment by Myshor** [comment]
Can any of you share the exact URL I could check?

I tried to play with some comments on old.reddit.com and could not reproduce any white backgrounds.

**Comment by victor6267** [comment]
So I'm seeing it on all reddit pages with comments. Below is a link to one
page I was on when I saw your email. I've also attached an image of my
current settings. I have *Theme Generation *set to Dynamic. The options
besides Dynamic do not cause comments to appear that way.
https://www.reddit.com/r/Helldivers/comments/1jydqii/is_there_a_way_to_program_shriekers_to_do_this_to/
[image: image.png]

**Reply by Myshor** [reply]
Still can not reproduce in Windows 11.

MS Edge and Dark Reader On:
![obraz](https://github.com/user-attachments/assets/6f17bd8b-9dfe-48ea-a194-061753bac89f)

Firefox and Dark Reader On:
![obraz](https://github.com/user-attachments/assets/9af549cc-42b0-42d9-ae78-cdef0f536398)

**Comment by victor6267** [comment]
My bad I didnt include my extension list. On reddit I do use RES with Dark
Mode disabled. I've found that subreddits with custom CSS styles turned on
do not have this issue but subreddits with that disabled will.
This subreddit has no custom styling so it has the issue: No time to cry :
KidsAreFuckingStupid
<https://www.reddit.com/r/KidsAreFuckingStupid/comments/1jyp9lf/no_time_to_cry/>
Then this page with styling ON has the correct child comment colors but
does not when it is turned off: "M'am, your sons died as heroes" :
Helldivers
<https://www.reddit.com/r/Helldivers/comments/1jyvtm0/mam_your_sons_died_as_heroes/>

**Reply by Myshor** [reply]
OK... I can confirm it sometimes uses white background when native dark mode on reddit is disabled.

Still... When you have native Dark Mode possible to use and it doesn't break with Dark Reader then why to disable native Dark Mode?

When you have native Light (I mean disabled Dark) then as the comments are loaded lazy (they load when you expand them) then the background is applied original from reddit.

To have it "fixed" you should turn off and turn back on Dark Reader so it would reapply the styles to the white loaded later parts.

I can additionally write it could not be fixed fast with sitefixes because of that lazy loading which forces original colors every time.
To fix this there must be some more analysis done some changes in the main code of Dark Reader.
Summarizing: it would be nice to have a bug submitted for this issue if you want to have it fixed as the discussions can be forgotten if nothing is changing here. 😉

**Comment by Sim0nV** [comment]
If u have RES, changing these settings fixed mine
https://github.com/darkreader/darkreader/issues/11895#issuecomment-3170476799
